app/apollo/index.js

```diff
@@ -126,7 +126,7 @@ const createApolloServer = () => {
     introspection: true, // set to true as long as user has valid token
     plugins: customPlugins,
     tracing: process.env.GRAPHQL_ENABLE_TRACING === 'true',
-    playground: process.env.GRAPHQL_ENABLE_PLAYGROUND === 'false',
+    playground: process.env.GRAPHQL_ENABLE_PLAYGROUND === 'true',
```

Comment:
why was this updated to true?

Reply:
I think this was a bug - the playground needs to be enabled when we export GRAPHQL_ENABLE_PLAYGROUND=true